src/event/index.ts

```diff
@@ -83,13 +83,17 @@ const connect = (scienceUUID: string, modhash: string) => {
 
 			voted.push(...toVote);
 			chrome.storage.local.set({ voted });
-			ws.send(
-				JSON.stringify({
-					type: "science",
-					uuid: scienceUUID,
-					upvoted: voted.length
-				})
-			);
+			chrome.storage.sync.get('scienceEnabled', (result) => {
```

Comment:
I'm not familiar with the style used for this repo, but I think this should be double quoted.